frontend/src/hooks/useProperties.ts

```diff
@@ -55,51 +50,12 @@ export const useProperties = () => {
     [dispatch, getPropertiesPaged],
   );
 
-  /**
-   * Make an AJAX request to fetch the specified 'property' from inventory.
-   * @param params unique id of the property
-   * @param position optional override for the lat/lng of the returned property.
-   */
-  const fetchProperty = useCallback(
-    async (id: number, position?: [number, number]): Promise<IProperty> => {
-      dispatch(logRequest(actionTypes.GET_PARCEL_DETAIL));
-      dispatch(showLoading());
-      // Due to spatial information being stored in BC Albers in the database, we need to make TWO requests here:
-      //   1. to the REST API to fetch property field attributes (e.g. address, etc)
-      //   2. to GeoServer to fetch latitude/longitude in expected web mercator projection (EPSG:4326)
```

Comment:
Did you check that this code is not user anywhere please? I recall putting this comment here recently to fix a bug in PSP...

Reply:
Yeah, removing this function did not cascade into any of the front end breaking